container/ubuntu16_04/builders/bazel/BUILD

```diff
@@ -38,6 +38,7 @@ toolchain_container(
         "//container/ubuntu16_04/layers/bazel:bazel-tools",
         "//container/ubuntu16_04/layers/bazel:bazel-ltl",
         "//container/ubuntu16_04/layers/git:git-ltl",
+        "//container/ubuntu16_04/layers/wget:wget-ltl",
```

Comment:
why not add wget to bazel-tools?

Reply:
I was considering that. The main reason I made it a separate LTL is because there is an Argo wget container (https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/wget). So I made it to live by itself. Same reason for the git LTL.

Reply:
sgtm